src/cloudant/document.py

```diff
@@ -153,7 +153,7 @@ def create(self):
         super(Document, self).__setitem__('_rev', data['rev'])
         return
 
-    def fetch(self):
+    def fetch(self, **kwargs):
```

Comment:
You'll need to add parameter descriptions for `kwargs` in this method's comment section.  See `db_updates(self, raw_data=False, **kwargs)` as an example.